src/Image/Image.php

```
@@ -9,23 +9,23 @@
 
 class Image
 {
-    const GRAVITY_CENTER = 'center';
+    public const string GRAVITY_CENTER = 'center';
```

PHPStan (2.1.17)

Class constants with native types are supported only on PHP 8.3 and later.


(classConstant.nativeTypeNotSupported)